### PR TITLE
Move VPN Resource Center images from Contentful CDN to Bedrock media

### PR DESCRIPTION
**Pull request by stevejalim**
_Do not merge for now_.

## One-line summary

This changeset moves the VPN RC away from using images live-hosted by Contentful. 

It:
* adds all of the images, in the appropriate size, from the Contentful CDN to our bedrock codebase
* contains a data migration that, when run, updates the `contentful_contentfulentry` table appropriately. (It swaps the `src` and/or `srcset` attributes of `img` tags from full Contentful CDN URLs to local image paths instead, matching the files added to the codebase. 
* EDIT: stops the regular syncing of Contentful content, else a change (or a forced update on Dev or a Demo) would overwrite the changes to use the local files

## Significant changes and points to review

The data migration is not pretty, but should make sense. Please call out anything that looks too strange. 

## Issue / Bugzilla link

No ticket

## Testing
Locally:

* on `main` run `./bin/run-db-download.py --force`
* switch to this branch 
* `./manage.py migrate` which should apply one migration (0007) in the contentful app
* Go to http://localhost:8000/en-US/products/vpn/resource-center/
   * Confirm there are no missing images, broken HTML, etc and that images are loaded from localhost not https://images.ctfassets.net
   * View each child page and confirm all OK and loading local images - having the Network tab in Devtools helps
   * When a child page contains an image, use devtools to drop the DPR from 2 to 1  and confirm via the Network tab that the lower-res image is loaded, and that it's locally sourced

### CODE REVIEW

**Review by alexgibson — APPROVED** [review]
I tested this locally and things seem to work as expected 👍 

I did notice a lot of the images are quite large, and we can save between 60% and 80% in file size by running things through [tinypng](https://tinypng.com/). It would be nice to do that before merging?

File: media/img/products/vpn/resource-center/image2_w1376.png
Comment (alexgibson) [inline media/img/products/vpn/resource-center/image2_w1376.png:1]: This image needs removing

File: media/img/products/vpn/resource-center/image2_w688.png
Comment (alexgibson) [inline media/img/products/vpn/resource-center/image2_w688.png:1]: This image also needs removing

File: media/img/products/vpn/resource-center/image1_w688.png
Comment (alexgibson) [inline media/img/products/vpn/resource-center/image1_w688.png:1]: This image shows ExpressVPN and should be removed

File: media/img/products/vpn/resource-center/image1_w1376.png
Comment (alexgibson) [inline media/img/products/vpn/resource-center/image1_w1376.png:1]: This image shows express VPN and should be removed

File: bedrock/contentful/migrations/0007_data_switch_to_local_images.py
Comment (robhudson) [inline bedrock/contentful/migrations/0007_data_switch_to_local_images.py:128]: Nicely done here. 👍

**Review by robhudson — APPROVED** [review]

**Comment by stevejalim** [comment]
@alexgibson Images optimised in https://github.com/mozilla/bedrock/pull/14198/commits/88d155b77a00199c240b55b2e78ca9c657538cae - thanks for the reminder!

**Comment by stevejalim** [comment]
Marking this as Do Not Merge, but could still do with a final review. It's ready to roll, but we may as well leave it until later in case we need to edit any copy. If we change images, this PR would need updating, but not copy changes.

**Review by alexgibson — APPROVED** [review]
r++

**Pull request merged**
